src/components/generic/StackedCardsSwiper.vue

```diff
@@ -136,8 +188,66 @@
   }
 
   .swiper-slide {
-    width: calc(200px + 12vmin);
+    width: calc(200px + 8vw);
+    max-width: 720px;
     height: auto;
+
+    @media (min-width: $bp-extraextralarge) {
+      width: calc(200px + 12vw);
+    }
+
+    h3 {
+      font-size: 1.25em;
+      font-weight: 500;
+      text-transform: uppercase;
+    }
+
+    p {
+      font-size: 1em;
+    }
+
+    --overlay: linear-gradient(0deg, rgba(0 0 0 / 60%), rgba(0 0 0 / 60%));
+
+    .card {
+      border: 0;
+      background-image: var(--overlay), var(--bg-img-small);
+      background-size: cover;
+      background-repeat: no-repeat;
+
+      @media (min-width: $bp-small) {
+        background-image: var(--overlay), var(--bg-img-medium, var(--bg-img-small));
+      }
+
+      @media (min-width: $bp-medium) {
+        background-image: var(--overlay), var(--bg-img-large, var(--bg-img-small));
+      }
+
+      @media (min-width: $bp-large) {
+        background-image: var(--overlay), var(--bg-img-xl, var(--bg-img-small));
+      }
+
+      @media (min-width: $bp-extralarge) {
+        background-image: var(--overlay), var(--bg-img-xxl, var(--bg-img-small));
+      }
+
+      @media (min-width: $bp-extraextralarge) {
```

Comment:
Let's adjust this, and possibly others, to avoid oversized images at certain resolutions. With the current queries, a 1920-width resolution window will receive the `xxlup`-size image, when it only needs the `xxl`.

When that is adjusted, I think some of the images are oversized for their respective breakpoints. I have found that these resolutions suffice:
* small: 245x440
* medium: 260x420
* large: 280x400
* xl: 300x400
* xxl: 450x400
* xxlup: 700x900